tasks/main.yml

```diff
@@ -38,5 +42,16 @@
     dest: /etc/sysconfig/kdump
     backup: yes
     mode: "{{ __sysconfig_kdump.stat.mode | d('0644') }}"
-  when: kexec_crash_size.content|b64decode|int > 0
   notify: restart kdump
+
+- name: Set the kdump_reboot_required fact
+  set_fact:
+    kdump_reboot_required: "{{ kexec_crash_size.content|b64decode|int < 1 }}"
```

Comment:
you could use `kdump_reboot_required: true` since the value is the same as the condition in the `when`

Reply:
I've fixed this by always setting this fact to either true or false